bulbs/content/tasks.py

```diff
@@ -34,3 +33,21 @@ def index_feature_type_content(featuretype_pk):
     featuretype = FeatureType.objects.get(pk=featuretype_pk)
     for content in featuretype.content_set.all():
         content.index()
+
+
+@shared_task(default_retry_delay=5)
+def update_feature_type_rates(featuretype_pk):
+    from bulbs.contributions.models import ContributorRole, Rate, FeatureTypeRate
+
+    roles = ContributorRole.objects.all()
+
+    for role in roles:
+        existing_rates = FeatureTypeRate.objects.filter(
+            feature_type_id=featuretype_pk,
+            role_id=role.pk)
+
+        if existing_rates.count() == 0:
```

Comment:
doesn't mater but this could alternatively be written as 
```if existing_rates.exists()``` and queries might even be false-y if they are empty naturally.